src/games/strategy/triplea/delegate/Matches.java

```diff
@@ -2016,27 +1947,20 @@ public boolean match(final Territory t) {
     };
   }
 
-  public static Match<Unit> unitIsEnemyAAforAnything(final PlayerID player, final GameData data) {
+  private static Match<Unit> unitIsEnemyAAforAnything(final PlayerID player, final GameData data) {
     final CompositeMatch<Unit> comp = new CompositeMatchAnd<Unit>();
     comp.add(UnitIsAAforAnything);
     comp.add(enemyUnit(player, data));
     return comp;
   }
 
-  public static Match<Unit> unitIsEnemyAAforCombat(final PlayerID player, final GameData data) {
+  private static Match<Unit> unitIsEnemyAAforCombat(final PlayerID player, final GameData data) {
     final CompositeMatch<Unit> comp = new CompositeMatchAnd<Unit>();
     comp.add(UnitIsAAforCombatOnly);
     comp.add(enemyUnit(player, data));
     return comp;
   }
 
-  public static Match<Unit> unitIsEnemyAAforBombing(final PlayerID player, final GameData data) {
```

Comment:
Might be worth keeping for future features, since `unitIsEnemyAAforBombing` mirrors its cousin, `unitIsEnemyAAforCombat`